buildSrc/src/main/java/Config.kt

```diff
@@ -1,7 +1,7 @@
 object Config {
     val agpVersion = "7.4.2"
-    val kotlinVersion = "1.9.21"
-    val composeVersion = "1.5.11"
+    val kotlinVersion = "1.9.23"
+    val composeVersion = "1.6.1"
```

Comment:
this is only temporary, with the current xcode version compiling with Kotlin <= 1.9.23 doesn't work